src/modules/cmd_duration.rs

```diff
@@ -79,12 +79,24 @@ fn undistract_me<'a, 'b>(
             unstyle(&ANSIStrings(&module.ansi_strings()))
         );
 
+        let timeout: u32 = match u32::try_from(config.notification_timeout) {
```

Comment:
I haven't tested the code, but I believe this can be written as something like the following:

```
let timeout = u32::try_from(config.notification_timeout).unwrap_or_else(|| {
  log::trace!("Configured notification timeout not within bounds. Defaulting to {} ms", v);
  CmdDurationConfig::default().notification_timeout as u32
})
```

which would be stylistically more similar to the rest of the codebase.

Reply:
Thanks, will give this a try after work.

Reply:
Latest commits add ModuleConfig for u32, which handles the checking and assigning of the default value instead of logic inside of cmd_duration.rs